tests/namespace_delete_test.go

```diff
@@ -82,16 +83,26 @@ func dynamicConfig() map[dynamicconfig.Key]interface{} {
 func (s *namespaceTestSuite) SetupSuite() {
 	s.logger = log.NewTestLogger()
 
-	clusterConfig, err := GetTestClusterConfig("testdata/integration_namespace_cluster.yaml")
-	s.Require().NoError(err)
+	var clusterConfig *TestClusterConfig
+	if TestFlags.PersistenceDriver == mysql.PluginNameV8 ||
+		TestFlags.PersistenceDriver == postgresql.PluginNameV12 ||
+		TestFlags.PersistenceDriver == sqlite.PluginName {
```

Comment:
Isn't `switch/case` easier to read?

Reply:
Yes, and in all other places we use `switch` for these constants also.